src/library/components/ProductCard.tsx

```diff
@@ -0,0 +1,163 @@
+    cardStyles = {...borderGhostStyle};
+  } else if (dropShadow) {
+    decorativeState = "bsds-card-shadow";
+    cardStyles = {...borderLightStyle};
+  }
+
+  useEffect(() => {
+    if(tag) {
+      setClonedTag(cloneElement(tag as ReactElement, {
+        isGhost: (variant === "ghost" || variant === "border-ghost"),
+     }));
+    }
+  },[tag, variant])
+
+  useEffect(() => {
+    if(image) {
+      setClonedImage(cloneElement(image as ReactElement, {
+        ratio: (image.props.ratio ? image.props.ratio : "1:1"),
+        isDecorative: false,
+        isGhost: (variant === "ghost" || variant === "border-ghost"),
+     }));
+    }
+  },[image, linkTo, variant])
+
+  useEffect(() => {
+    //This is to expand the link hitbox for 50% split image
+    if(image?.props.ratio === "50:50") {
+      const productCardLinks = document.querySelectorAll(".bsds-product-card-with-image");
+      Array.prototype.forEach.call(productCardLinks, productCardLink => {
+        let keyDown:number, keyUp:number, link = productCardLink.querySelector("a");
+        productCardLink.onmousedown = () => keyDown = +new Date();
+        productCardLink.onmouseup = () => {
+          keyUp = +new Date();
+          if((keyUp - keyDown) < 200) {
+            link.click();
+          }
+        }
+      });
+    }
+  }, [image?.props.ratio])
+
+  const defaultProductCard = (
+    <div className={decorativeState ? `${"bsds-product-card"} ${cardStyles.classes} ${decorativeState}` : `${"bsds-product-card"} ${cardStyles.classes}`} data-testid="bsdsProductCard">
+      { image && clonedImage }
+      { tag && clonedTag }
+      { headingLevel ? <HeadingElement headingLevel={headingLevel} className="bsds-card-title" id={"productTitle" + productNameId}>
+        <Link href={linkTo} className={`${cardStyles.titleLinkClasses}`}>
+          { texts.title }
+        </Link></HeadingElement> : <Link href={linkTo} className={`${"bsds-product-card-ns-title"} ${cardStyles.titleLinkClasses}`} id={"productTitle" + productNameId}>
+          { texts.title }
+      </Link> }
+      <p className="bsds-card-description">{texts?.description}</p>
+    </div>
+  )
+
+  if (image?.props.ratio === "50:50") {
+    return (
+      <div data-testid="bsdsProductCard" className="bsds-even-split-image-product-card">
+        { image && clonedImage }
+        <div className={decorativeState ? `${"bsds-product-card"} ${cardStyles.classes} ${decorativeState}` : `${"bsds-product-card"} ${cardStyles.classes}`}>
+          { tag && clonedTag }
+          { headingLevel ? <HeadingElement headingLevel={headingLevel} className="bsds-card-title" id={"productTitle" + productNameId}>
+            <Link href={linkTo} className={`${cardStyles.titleLinkClasses}`}>
+              { texts.title }
+            </Link></HeadingElement> : <Link href={linkTo} className={`${"bsds-product-card-ns-title"} ${cardStyles.titleLinkClasses}`} id={"productTitle" + productNameId}>
+              { texts.title }
+          </Link> }
```

Comment:
Can we break these element onto their own lines?
```jsx
{ headingLevel ? (
  <HeadingElement>
    <Link>
      { texts.title }
    </Link>
  </HeadingElement>
) : (
  <Link>
    { texts.title }
  </Link>
)
```

You may or not need the parens.